src/main.cpp

```diff
@@ -16,43 +18,46 @@ int main(int argc, char *argv[])
 
   const auto start = std::chrono::high_resolution_clock::now();
 
-  bool flagwarm;
-  int resolution;
-  double zmax;
-  double nu1, nu2, G1, G2, E, alpha, k_el, delta, nnodi, to1, E1, E2, lato, errf, Delta;
-  bool rmg_flag;
-  bool rand_seed_flag;
+  bool WarmStartingFlag;
+  int Resolution;
+  double MaxTopologyHeight;
+  double nu1, nu2, CompositeYoungs, alpha, ElasticComplianceCorrection, GridSize, Tolerance, E1, E2,
+      LateralLength, Delta;
+  bool RandomTopologyFlag;
+  bool RandomSeedFlag;
   double Hurst;
-  std::string zfilePath;
-  int rmg_seed;
-  int max_iter;
+  std::string TopologyFilePath;
+  int RandomGeneratorSeed;
+  int MaxIteration;
 

```

Comment:
It's good practice to initialize variables right at declaration. If not, you rely on `MIRCO::SetParameters()` to properly initialize all variables. Of course, setting the desired parameter from input should happen in `MIRCO::SetParameters()`, but initialization could happen before (so that you have maybe wrong values, but at least _a_ value for each variable.